.rubocop.yml

```diff
@@ -6,7 +6,7 @@ AllCops:
     - vendor/**/*
     - examples/**/*
 
-Metrics/LineLength:
+Layout/LineLength:
```

Comment:
To fix warning: `.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout` when running rubocop.